models/intermediate/field_history/int_jira__daily_field_history.sql

```diff
@@ -28,8 +28,9 @@ limit_to_relevant_fields as (
 
     from combined_field_histories
 
-    where lower(field_id) = 'status' -- As sprint is a custom field, we filter by field name only for sprint. All others are on field_id.
-        or lower(field_name) in ('status'
+    where lower(field_name) in 
+            ('status'
+            ,'sprint'
```

Comment:
Make sense, Thanks for the review! Reverted `status` to be checked against `field_id` as recommended

Reply:
I did this as a seperate commit (instead of accepting the change) in order to update `models/intermediate/field_history/int_jira__pivot_daily_field_history.sql` at the same time